shotover-proxy/src/transforms/cassandra/sink_cluster/mod.rs

```diff
@@ -344,38 +346,52 @@ impl CassandraSinkCluster {
                         );
                     }
 
-                    // This is purely an optimization: To avoid opening these connections sequentially later on, we open them concurrently now.
-                    try_join_all(
+                    // This is purely an optimization:
+                    // To avoid opening these connections sequentially later on, we open them concurrently now.
+                    // We ignore errors, the code that actually uses the connection is responsible for reporting the errors
+                    let _ignore_errors = join_all(
```

Comment:
We could just `warn!` with these errors

Reply:
Oh yes, thinking about it more you are right.
It would be consistent with the other handling to do a `node.report_issue()` and a warning.